develop-docs/sdk/javascript/browser-tracing.mdx

```diff
@@ -0,0 +1,151 @@
+---
+
+The tracing behavior in our browser SDKs is somewhat unique and significantly differs from tracing in the backend. This page collects the most important aspects.
+
+<Note>
+  The aspects described in this document apply to all browser SDKs. This
+  includes `@sentry/browser` and all SDKs building on top of it like
+  `@sentry/react`, `@sentry/angular` or `@sentry/vue` but also the client-side
+  parts of meta frameworks like `@sentry/nextjs`.
+</Note>
+
+Please note that any kind of automatic tracing instrumentation in the browser requires the `browserTracingIntegration()` to be added to the SDK configuration.
+The default configuration of the SDK does not include any performance or tracing-related instrumentation to save on [bundle size](../bundle-size) for errors-only users.
+
+## Pageload and Navigation Spans
+
+Browser SDKs automatically create spans for the initial pageload as well as for subsequent soft navigations.
+
+These spans however don't behave like conventional spans. We call them **idle spans** because they are started at some point (more on that later) but not explicitly ended.
+Instead, we use a couple of heuristics and parameters to debounce and end the spans automatically.
+For example, we end an idle span after a certain time of inactivity, meaning no new spans are added to the span tree.
+Also, there are maximum timeout parameters.
+This idling mechanism is necessary because neither in pageload nor navigation scenarios we have a clear end point or event to explicitly end the span.
+It's important to understand this behavior to interpret the duration of the root span in a sensible way:
+
+While at the end of the idle span, we shorten its duration to the longest child span, the duration does not necessarily represent the actual time a pageload or navigation took.
+Due to the debounce mechanism, child spans that you'd intuitively not associate to the initial pageload might be created close enough to the ideal end of the pageload span to prolong it.
+
+Both, pageload and navigation spans are **always started as root spans**.
+
+### Pageload Spans
+
+The pageload span is started when the SDK initializes, more specifically, when the `browserTracingIntegration`'s `afterAllSetup` hook is called. This ensures that we start the
+pageload span as early as possible. However, this still means that the span is only started once the SDK code is shipped to the browser. In reality though, the actual page load
+starts with the browser making the request for the URL to the server. To account for this, we retroactively backdate the start time of the pageload span to the browser request
+start time. We are able to backdate the start time because the browser provides this timing information.
+
+The pageload span is **earliest** ended, when the browser emits when `'interactive'` or `'complete'` in `WINDOW.document.readyState`. It might be very well prolonged by our instrumentation adding additional child spans to the pageload root span.
```

Comment:
```suggestion
The pageload span is **earliest** ended, when the browser emits `'interactive'` or `'complete'` in `WINDOW.document.readyState`. It might be very well prolonged by our instrumentation adding additional child spans to the pageload root span.
```